src/main/java/org/gridsuite/boundary/importer/job/BoundaryAcquisitionServer.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.boundary.importer.job;
+
+import org.apache.commons.vfs2.FileObject;
+import org.apache.commons.vfs2.FileSystemException;
+import org.apache.commons.vfs2.FileSystemOptions;
+import org.apache.commons.vfs2.auth.StaticUserAuthenticator;
+import org.apache.commons.vfs2.impl.DefaultFileSystemConfigBuilder;
+import org.apache.commons.vfs2.impl.StandardFileSystemManager;
+import org.apache.commons.vfs2.provider.ftp.FtpFileSystemConfigBuilder;
+import org.apache.commons.vfs2.provider.sftp.SftpFileSystemConfigBuilder;
+import org.gridsuite.boundary.importer.job.utils.CgmesBoundaryUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+public class BoundaryAcquisitionServer implements AutoCloseable {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(BoundaryAcquisitionServer.class);
+
+    private final StandardFileSystemManager fsManager = new StandardFileSystemManager();
+    private FileSystemOptions fsOptions = new FileSystemOptions();
+
+    private String serverUrl;
+
+    public BoundaryAcquisitionServer(String url, String userName, String password) throws FileSystemException {
+        serverUrl = url;
+
+        StaticUserAuthenticator auth = new StaticUserAuthenticator(null, userName, password);
+        DefaultFileSystemConfigBuilder.getInstance().setUserAuthenticator(fsOptions, auth);
+
+        SftpFileSystemConfigBuilder.getInstance().setUserDirIsRoot(fsOptions, true);
+        SftpFileSystemConfigBuilder.getInstance().setPreferredAuthentications(fsOptions, "publickey,password");
+        SftpFileSystemConfigBuilder.getInstance().setConnectTimeoutMillis(fsOptions, 30000);
```

Comment:
Add a constant ?

Reply:
or a parameters, some server a slow to respond?

Reply:
Constant added